.github/stale.yml

```diff
@@ -14,6 +14,7 @@
 
 staleLabel: stale
 daysUntilClose: 14
```

Comment:
we should also move this into the pulls section, just in case.

Reply:
I can do that; but it seems weird to me that moving it inside pulls affects issues as well; seems more logical if its outside pulls as then it will have a global effect, no ?